docs/docs/end-to-end-testing.md

```diff
@@ -49,10 +51,125 @@ If you want to run Cypress in Continuous Integration (CI) you have to use `cypre
   "scripts": {
     "develop": "gatsby develop",
     "cy:open": "cypress open",
-    "cy:run": "cypress run",
-    "test:e2e:ci": "start-server-and-test develop http://localhost:8000 cy:run"
+    "cy:run": "cypress run", // highlight-line
+    "test:e2e": "start-server-and-test develop http://localhost:8000 cy:open"
+    "test:e2e:ci": "start-server-and-test develop http://localhost:8000 cy:run" // highlight-line
   }
 }
 ```
 
 Please read the [Cypress' official documentation](https://docs.cypress.io/guides/guides/continuous-integration.html) on CI if you want to know how to setup Travis or GitLab with Cypress.
+
+## Writing tests
+
+The first test is an **accessibility** test with [cypress-axe](https://github.com/avanslaars/cypress-axe).
```

Comment:
I feel like I should know the answer to this, but what do you mean by "the first test"?

Reply:
I mean that is the first test someone will write by reading this page.